.github/workflows/linter-irods-clang-tidy.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
                 # Run clang-tidy on the changes.
                 git diff -U0 origin/$GITHUB_BASE_REF | \
-                    clang-tidy-diff.py -p1 -path build/compile_commands.json -quiet | \
+                    clang-tidy-diff.py -p1 -path build/compile_commands.json -quiet -timeout 900 | \
```

Comment:
The clang-tidy workflow for irods/irods uses a timeout of 300 seconds. Is there a reason this workflow uses a different value?

Reply:
It's to account for PREP, which can still take a while to compile.

Reply:
Got it. Please leave a comment stating that so it's clear to the next person who lands here.

Reply:
Done